src/entry_point/start_item_riscv32.rs

```diff
@@ -50,8 +50,7 @@ pub unsafe extern "C" fn _start() -> ! {
     // Otherwise after the first syscall (the memop to set the brk), the return
     // will use a stack that is outside of the process accessible memory.
     //
-    add t2, t0, t1          // t2 = stacktop + appdata_size
-    bgt t2, a3, skip_set_sp // Compare `app_heap_break` with new brk.
+    bgt t1, a3, skip_set_sp // Compare `app_heap_break` with new brk.
```

Comment:
Maybe make it clear that `t1` is `appdata_size` in the comments?

Reply:
Ah I thought I committed that, I'll fix it.